pkg/db/db.go

```diff
@@ -80,14 +80,11 @@ func Init(dbDir string, opts ...Option) (err error) {
 	eb = eb.With("db_path", dbPath)
 
 	// bbolt sometimes occurs the fatal error of "unexpected fault address".
-	// In that case, the local DB should be broken and needs to be removed.
```

Comment:
The database needs to be removed. Otherwise, Trivy always fails due to the broken database.

Reply:
hm... it make sense
updated in https://github.com/aquasecurity/trivy-db/pull/501/commits/6100c2e3d9837c28b1f7f88d5e7c5358d4c258c1

Reply:
Yes. I got it right away.
It will also help users - they won't have to manually delete the broken database.